topics/ecology/tutorials/panoply_ebv/tutorial.md

```diff
@@ -24,7 +24,10 @@ contributors:
 - colineroyaux
 - Marie59
 - annefou
-
+funding:
```

Comment:
you'll need to make the key about `contributions:` and add an `authorship:` for the four of you

Reply:
thanks!